doc/uml/mtaosClass.uml

```diff
@@ -1,6 +1,6 @@
 @startuml
 Model *-- Config
```

Comment:
Based on the code, this relation has been removed. `Model` class does not hold the `Config` object as an attribute anymore.